source/playbooks/NIST80053/ssmdocs/descriptions/AutoScaling.1.md

```diff
@@ -0,0 +1,16 @@
+### Document Name - ASR-AFSBP_1.0.0_AutoScaling.1
```

Comment:
None of these descriptions should be necessary unless you're defining the document here rather than using an identical copy from SC.